### PR TITLE
Add AbhijithGanesh to the list

### PR DESCRIPTION
**Pull request by AbhijithGanesh**
I've added my profile which will reflect on the linkfree website.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/277"></a>

### CODE REVIEW

File: public/data/AbhijithGanesh.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+            "url": "https://linkedin.com/in/AbhijithGanesh14",
+            "icon": "linkedin"
+        },
+        {
+            "name": "Check my website out",
+            "url": "https://AbhijithGanesh.tech",
+            "icon": "web"
+        }
+    ]
+}
```
Comment (Panquesito7) [inline public/data/AbhijithGanesh.json:22]: Missing an endline.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by AbhijithGanesh** [comment]
Done Panquesito

File: public/data/AbhijithGanesh.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+            "url": "https://github.com/AbhijithGanesh",
+            "icon": "github"
+        },
+        {
+            "name": "Conenct with me on LinkedIn",
+            "url": "https://linkedin.com/in/AbhijithGanesh14",
+            "icon": "linkedin"
+        },
+        {
+            "name": "Check my website out",
```
Comment (Panquesito7) [inline public/data/AbhijithGanesh.json:17]: What about
```suggestion
            "name": "Check out my website",
```

**Comment by AbhijithGanesh** [comment]
Done.

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**